meresco/dans/longconverter.py

```diff
@@ -532,7 +528,19 @@ def _getRelatedIdentifiers(self, lxmlNode, e_longmetadata):
                         attrib_dict['type'] = relId.get_name()
                         attrib_dict['idx'] = relId.get_idx_id()
                         etree.SubElement(e_longmetadata, "related_identifier", attrib_dict).text = relId.get_unformatted_id()
-
+        elif self._metadataformat.isDC():
+            identifierList = lxmlNode.xpath("//dc:relation/text()", namespaces=namespacesmap)
+            for identifier in identifierList:
+                attrib_dict = {}
+                pidtype, pidvalue = self._getOpenAIREIdentifierFromDcRelation(identifier, "reference|dataset")
+                pId = PidFactory.factory(pidtype, pidvalue)
+                if pId.is_valid():
+                    attrib_dict['type'] = pId.get_name()
+                    attrib_dict['idx'] = pId.get_idx_id()
+                    attrib_dict['relationType'] = "References"
+                    if str(identifier).find("semantics/dataset/") >= 0:
```

Comment:
When we sat together to look at this issue, I understood that we came to a conclusion that if the user writes an identifier in 'semantics/dataset' s/he really means that it is a dataset the identifier is referencing to.  But we also concluded that if the user writes an identifier in 'semantics/reference' it could be that it is not a publication s/he is referring to, even though according to specs only publication is a valid reference.

Reply:
resourceTypeGeneral  `publication` added for identifiers in `semantics/reference`